src/routes/root/Summary.tsx

```diff
@@ -77,13 +75,20 @@ export function Summary(): JSX.Element {
       );
 
       const gas = await depositMethod.estimateGas({ value: value.toString() });
-      const result = await depositMethod.send({
-        from: accounts[0],
-        gas: gas.toString(),
-        value: value.toString(),
-      });
-      console.log(result);
-      navigate('/transactions');
+      await depositMethod
+        .send({
+          from: accounts[0],
+          gas: gas.toString(),
+          value: value.toString(),
+        })
+        .on('receipt', (receipt: TransactionReceipt) => {
```

Comment:
You should listen on the Deposit event of the bridge contract here otherwise replacing transaction will screw the flow^^